pkg/destroy/aws/aws.go

```diff
@@ -183,7 +185,17 @@ func (o *ClusterUninstaller) Run() error {
 
 									err = deleteARN(awsSession, parsed, filter, arnLogger)
 									if err != nil {
-										arnLogger.Debug(err)
+										if firstSeen, ok := errorsSeen[arnString]; ok {
+											if time.Now().Sub(firstSeen) > 5*time.Minute {
```

Comment:
https://golang.org/pkg/time/#Since

Reply:
Ah, missed that one... I only looked at the Time.* functions. Done.